src/js/components/Layout/menu/MenuItem.jsx

```diff
@@ -9,7 +9,7 @@ const MenuItem = ({ section, key, active }) => (
       {section.label}
     </a>
     <div className="dropdown-menu" aria-labelledby="navbarDropdown">
-      <div style={{ maxHeight: '60vh', overflow: 'auto' }} className="px-3 py-1">
+      <div className="padding-8">
```

Comment:
I also realized that while doing React menu I didn't realize that I hadn't changed the class for this `div` and as result, the Stocklist's dropdown had more padding that it should have, so I also fixed it.